packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.tsx

```diff
@@ -1,64 +1,74 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Formik, Form, Field } from 'formik';
+import { Formik, Form, Field, FieldProps, FormikProps } from 'formik';
 import { Input, Button } from '@deriv/components';
-import { getPropertyValue, WS } from '@deriv/shared';
+import { useSendUserOTP } from '@deriv/api';
 import { localize } from '@deriv/translations';
+import { useStore } from '@deriv/stores';
 
-const DigitForm = ({ is_enabled, setTwoFAStatus, setTwoFAChangedStatus, is_language_changing }) => {
-    const [is_success, setSuccess] = React.useState(false);
-    const [is_ready_for_verification, setReadyForVerification] = React.useState(false);
-    const button_text = is_enabled ? localize('Disable') : localize('Enable');
-    const formik_ref = React.useRef();
-    let enable_response;
+type TDigitFormValues = {
+    digit_code: string;
+};
+
+const DigitForm = () => {
+    const { client, common } = useStore();
+    const { is_language_changing } = common;
+    const { has_enabled_two_fa, setTwoFAChangedStatus, setTwoFAStatus } = client;
+
+    const { is_TwoFA_enabled, error, isSuccess, sendUserOTP } = useSendUserOTP();
+    const button_text = has_enabled_two_fa ? localize('Disable') : localize('Enable');
```

Comment:
can this be `Localize` and passed to `Button` as `children`?

Reply:
Actually no because`TButtonCommonProps` of `Button` component takes only string type for text prop